RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+25.0
+-----
+* [*] Site picker: Fixed the UI alignment issue in RTL [https://github.com/wordpress-mobile/WordPress-Android/pull/20804]
```

Comment:
Due to timezones I think we have a few more hours [till the release 24.9 is cut](https://github.com/wordpress-mobile/WordPress-Android/tree/release/24.9). Wdyt of targeting 24.9 and updating the release note?

Reply:
Sounds good!